librz/core/cmd_descs/cmd_debug.yaml

```diff
@@ -459,8 +587,31 @@ commands:
         subcommands: cmd_heap_glibc
       - name: dmi
         summary: List/Load symbols
-        cname: cmd_debug_dmi
-        type: RZ_CMD_DESC_TYPE_OLDINPUT
+        subcommands:
+          - name: dmi
+            summary: List symbols of target lib
+            cname: cmd_debug_dmi
+            args:
+              - name: libname
+                type: RZ_CMD_ARG_TYPE_STRING
+                optional: true
+              - name: symname
+                type: RZ_CMD_ARG_TYPE_STRING
+                optional: true
+            modes:
+              - RZ_OUTPUT_MODE_STANDARD
+              - RZ_OUTPUT_MODE_RIZIN
+              - RZ_OUTPUT_MODE_QUIET
+              - RZ_OUTPUT_MODE_JSON
+              - RZ_OUTPUT_MODE_LONG
+          - name: dmi.
+            summary: List closest symbol to the current address
+            cname: cmd_debug_dmid
+            args: []
+      #          - name: dmiv
```

Comment:
https://github.com/rizinorg/rizin/blob/b3de5151537aad66aad213caf1443e46c4861394/librz/core/cmd/cmd_debug.c#L1232-L1239

Because they were processed together in the original, and then it still use `RZ_MODE_SIMPLE`, `RZ_MODE_JSON`, but now we seem to use `RzOutputMode`.
And I haven't completely changed this part `dmi`.

Reply:
`dmiv` seems to be just an alias for `dmm`